dspace/config/dspace.cfg

```diff
@@ -1569,6 +1569,16 @@ log.report.dir = ${dspace.dir}/log
 # The max number of events held in the GA buffer (default: 256)
 # google.analytics.buffer.limit=256
 
+####################################################################
+#------------------- ADS SERVICES ---------------------------------#
+#------------------------------------------------------------------#
+submission.lookup.ads.key =
+####################################################################
+#------------------- EPO SERVICES ---------------------------------#
+#------------------------------------------------------------------#
+epo.consumerKey =
+epo.consumerSecretKey =
```

Comment:
These should all be moved to the new config you created in your other PR.  They also need comments to describe them.  In your comments, please make sure to describe what "ADS" means and what "EPO" means... abbreviations are not always easy to understand.